geopandas-guide/geoEDA_README.md

```diff
@@ -0,0 +1,47 @@
+# Getting Started With GeoPandas
+
+## Description
+This Jupyter Notebook performs a geospatial analysis focusing on the WiFi locations around the city of Boston using the GeoPandas library. It demonstrates how to visualize, clean, and manipulate geospatial data effectively.
+
+## Datasets
+
+- 2020 CENSUS TRACTS IN BOSTON dataset, which contains the geographical boundaries of each census tract in Boston.
```

Comment:
can you include the links to these datasets from analyze boston?

Reply:
yes done